scripts/compare-features.js

```diff
@@ -29,6 +31,12 @@ const compareFeatures = (a, b) => {
       if (capsWordA) return -1;
       if (capsWordB) return 1;
     }
+    if (a == 'secure_context_required') return -1;
```

Comment:
nit: empty line to separate the block

```suggestion

    if (a == 'secure_context_required') return -1;
```